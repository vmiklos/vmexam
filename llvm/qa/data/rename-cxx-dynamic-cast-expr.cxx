class C
{
public:
    int getValue() const
    {
        return 0;
    }
};

int main()
{
    C* pC = new C();
    dynamic_cast<const C&>(*pC).getValue();
    dynamic_cast<const C*>(pC)->getValue();
}
