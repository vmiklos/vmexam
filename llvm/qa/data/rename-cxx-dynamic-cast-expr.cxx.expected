class D
{
public:
    int getValue() const
    {
        return 0;
    }
};

int main()
{
    D* pC = new D();
    dynamic_cast<const D&>(*pC).getValue();
    dynamic_cast<const D*>(pC)->getValue();
}
